install

```diff
@@ -4,14 +4,14 @@
 #return with a non-zero return value.
 
 if [[ $EUID -ne 0 ]]; then
-    echo "Please run as root user."
-    exit 1
+  echo "Please run as root user."
+  exit 1
 fi
 
-SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
+SCRIPT_DIR="$( cd "$( dirname "$0")"  && pwd)"
```

Comment:
Inconsistent spacing here, the ` ` after the opening bracket should be removed too